m2py/scripts/example.py

```diff
@@ -1,252 +0,0 @@
-import argparse
```

Comment:
I haven't been updating this script. The jupyter notebook with current capabilities is its replacement